google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/GqlQuery.java

```diff
@@ -291,6 +291,16 @@ static Binding fromPb(com.google.datastore.v1.GqlQueryParameter argPb) {
       return this;
     }
 
+    /**
+     * Sets a new named binding with null value.
+     *
+     * @param name of the binding
+     */
+    public Builder<V> setBinding(String name) {
```

Comment:
Please rename this method to `setNullBinding` so that the name of the method matches the behavior that is implemented.

Reply:
Done. PTAL